docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -142,13 +147,13 @@ tcld apikey create --name Your-Key-Name --description YourDescription --duration
 
 :::note
 
-Use tcld to create API Keys with a day level expiration by day.
+Use tcld to create API Keys with a day level expiration.
 
 :::
 
 ### Delete an API Key
 
-Delete an API Key using one of the following methods.
+Delete an API Key using the Cloud UI or tcld. Deleting a key removes the ability for that API Key to authenticate into Temporal Cloud. Be aware that if you delete an API Key for a running WF, that WF will fail unless you rotate the API Key for a new one.
```

Comment:
I would change the abbreivation here too!

Reply:
We have not yet resolved a peer consensus about one sentence per line on docs. This practice allows cleaner diffs and feedback for revision. Happy to go along with whatever the team decides on.

Don't use WF. We use full words like Workflow.

Suggested rewording: 

Deleting an API key deactivates its permissions to authenticate into Temporal Cloud.
Please note that this affects running Workflows that were initialized with a deleted API key except when the API key was rotated into place.
This can affect long-running Workflows.

And then add a link to the API key rotation process in-text.

Reply:
Good catch. Agreed that this doc should have one sentence per line. Otherwise it might end up being a  correction afterwards.